File: MODELJUNIT/src/implementation/profonline/models/classRecord.php
<?php
  class ClassRecord {
    private $listPlan = [ '0' => ['period' => 20192, 'classId' => 1, 'hours' => 60 ], '1' => ['period' => 20192, 'classId' => 2, 'hours' => 30] ];

    public function getHours( $period, $classId ){
      foreach ($this->listPlan as $key => $value) {
        // $arr[3] será atualizado com cada valor de $arr...
        if( $period == $value['period'] && $classId == $value['classId'] ){
          echo $value['hours'] . "";die;
        }
      }
      echo 0;
    }

    public function insertClassRecord( $period, $classId, $hours, $contentClass ) {

      foreach ($this->listPlan as $key => $value) {
        // $arr[3] será atualizado com cada valor de $arr...
        if( $period == $value['period'] && $classId == $value['classId'] && $contentClass != '' ){
          echo true;die;
        }
      }
      echo false;
      //buscar na lista toda um valor e retornar o hours...
    }
  }
?>

File: Sd2Stubs/Files/ClassRecord.java
import java.util.Date;

public class ClassRecord {
	private String period = "20192";  
	private String classId = "1";  
	private int hours = 60;  

	public String getPeriod() {
		return period;
	}  

	public String getClassId() {
		return classId;
	}  

	public int getHours(String period, String classId){
		return hours;
	}  

	public boolean insertClassRecord(String period, String classId, Date date, int hours, String contentClass) {
		if (this.period.equals(period) && this.classId.equals(classId) && hours > 0 &&  contentClass != ""){
			return true;
		}
		else{
			return false;
		}
	}
}
